tox.ini

```diff
@@ -13,4 +13,4 @@ deps =
     nose
     django_nose
     six
-    django22: Django>=2.2, <2.3
+    django22: Django>=2.2
```

Comment:
This will install django 3 something. Which means `django22` is no longer a valid name.

Reply:
Right, `django22` should be constrained to the latest version of django2.2.